<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadExampleDialog</class>
 <widget class="QDialog" name="LoadExampleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,5,0">
   <item row="2" column="0">
    <widget class="QPushButton" name="buttonLoad">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="FluidFlowView" name="viewWidget" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QListWidget" name="examplesListView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FluidFlowView</class>
   <extends>QWidget</extends>
   <header>FluidFlowView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
